### ISSUE TITLE
Logging overwrite less sweeping

### ISSUE THREAD

**Issue by jamesmf** [zalandoresearch/flair#415]
To be removed, once it is done: Please add the appropriate label to this ticket, e.g. feature or enhancement.

**Is your feature/enhancement request related to a problem? Please describe.**
When using flair in other applications, the fact that it disables existing logs in `__init__.py` can be detrimental. For instance when wrapping it up as a component in a tool like rasa_nlu, importing flair overrides all logging except its own, breaking functionality in rasa_nlu.

This is the [line that does so ](https://github.com/zalandoresearch/flair/blob/c2bb0d8776f25493a5b994dcd89a96f71ac175b8/flair/__init__.py#L13) and it was done on purpose to disable BERT logging in #282 .

**Describe the solution you'd like**
Ideally, the problem of disabling logging from certain known dependencies should be much more limited in scope. Importing flair as a package shouldn't disable all the other loggers. At a minimum, perhaps the logging could only *optionally* disable all existing logs

**Comment by jamesmf** [comment]
Also worth noting re: #282  it seems like the issue was solved upstream

ace/pytorch-pretrained-BERT/issues/117

**Comment by alanakbik** [comment]
Thanks for pointing this out! 

@tabergma could you take a look?

**Comment by dennisverspuij** [comment]
I personally think the logging config should be removed altogether, just like done in [pytorch-pretrained-BERT#117](https://github.com/huggingface/pytorch-pretrained-BERT/issues/117). It is a choice of user of the library to have logging or not, and how. Can the logging setup be made a function or so, that sets up loggin if not already done so, or augments existing logging configuration? I now have to import flair at the top, remove handlers afterwards to be able to use my own logging configuration.

**Comment by antocapp** [comment]
@alanakbik I still have problems in logging and can't log my own logger

**Comment by alanakbik** [comment]
@antocapp could you paste a minimal code example that illustrates the specific problem you have so we can reproduce?

**Comment by danielchalef** [comment]
Flair still forces the root logger to `WARN` (see https://github.com/zalandoresearch/flair/blob/master/flair/__init__.py#L39), overriding any user provided basicConfig.

A minimal example:
```
import logging
logging.basicConfig(
    format="%(asctime)s : %(levelname)s : %(message)s", level=logging.INFO
)
logging.info("First info")

import flair
logging.info("Second info")
```
Only the first log message will be output to the console.

**Comment by alanakbik** [comment]
@danielchalef thanks for posting the example! I can remove this line from the init.py, then the example works again. Or is there a better way to fix this?

**Comment by danielchalef** [comment]
@alanakbik if the upstream logging issue has been resolved, it would make sense to remove the root config.
